endorsement/dao/notification.py

```diff
@@ -310,6 +310,47 @@ def warn_endorsers(notice_level, lifetime):
                 pass
 
 
+def create_warn_shared_owner_message(owner_netid, endorsements):
+    context = {
```

Comment:
You're checking that endorsements is not None and has a len > 0 below - I'd do that here or add an underscore to signify that this is a private method.

Reply:
there's a few other procedures in that pattern that could be adjusted